cmd/server/main.go

```diff
@@ -46,7 +53,7 @@ func main() {
 	if AuthzAddr != "" {
 		interceptors = append(interceptors,
 			// authorization interceptor
-			toolkit_auth.DefaultAuthInterceptor(AuthzAddr),
+			toolkit_auth.DefaultAuthInterceptor(AuthzAddr, applicationID),
```

Comment:
should be toolkit_auth.UnaryServerInterceptor();
also corresponding import should match latest version of toolkit